### PR TITLE
Organize sidebars.js

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
#### Did you test the changes locally ?
- [ ] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/proxy.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 id: proxy
-title: Proxy
+title: Proxy architecture
 ---
 
 Overview of the MultiversX Proxy
 
+## **Introduction**
+
+Proxy is a go project that abstracts the sharding logic. It is the project backing *gateway.elrond.com*
```
Comment (schimih) [inline docs/sdk-and-tools/proxy.md:10]: maybe:
```suggestion
Proxy is a project written in go that abstracts the sharding logic. It servers as foundation for *gateway.elrond.com*
```

**Review by schimih — APPROVED** [review]

File: docs/sdk-and-tools/proxy.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
 id: proxy
-title: Proxy
+title: Proxy architecture
 ---
 
 Overview of the MultiversX Proxy
 
+## **Introduction**
+
+Proxy is a project written in go that abstracts the sharding logic. It serves as foundation for *gateway.elrond.com*
```
Comment (miiu96) [inline docs/sdk-and-tools/proxy.md:10]: gateway.multiversx.com

File: docs/validators/nodes-scripts/install-update.md [hunk 3]
```diff
@@ -5,16 +5,20 @@ title: Installing a Validator Node
 
 ## **Install your node(s)**
 
-After preparing the user permissions, the script configurations, and the keys, the actual node installation can begin. The Validator script is a multi-purpose tool for managing your node, it is accessible to Devnet and Testnet Network.
-
-- _Note: This process installs a validator node for all MultiversX networks (mainnet, devnet, and testnet)._
+After preparing the user permissions, the script configurations, and the keys, the actual node installation can begin. The Validator script is a multi-purpose tool for managing your node, it is accessible to Mainnet, Devnet or Testnet Network.
```
Comment (miiu96) [inline docs/validators/nodes-scripts/install-update.md:8]: can you remove `Network` word from the end of the sentence?

File: docs/sdk-and-tools/proxy.md (hunk 2 above)
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/proxy.md]: done

File: docs/validators/nodes-scripts/install-update.md (hunk 3 above)
Comment (bogdan-rosianu) [reply docs/validators/nodes-scripts/install-update.md]: done

**Review by schimih — APPROVED** [review]

**Pull request merged**